corehq/apps/app_manager/static/app_manager/js/details/column.js

```diff
@@ -100,8 +100,12 @@ hqDefine("app_manager/js/details/column", function () {
                 label: gettext('Cache & Lazy Load'),
             },
         ];
-        self.optimizationSelectElement = uiElementSelect.new(optimizationOptions).val(self.original.optimization || "");
-        self.$optimizationSelectElement = $('<div/>').append(self.optimizationSelectElement.ui);
+        if (screen.showCaseListOptimizations) {
```

Comment:
to limit the impact of this PR to only when applicable and reduce the impact radius.